usaspending_api/download/management/commands/populate_monthly_files.py

```diff
@@ -49,9 +49,8 @@ def download(
             date_range["end_date"] = end_date
         json_request = {
             "constraint_type": "year",
-            "award_levels": award_levels,
             "filters": {
-                "award_types": award_types,
+                "prime_award_types": prime_award_types,
```

Comment:
Will this ever need the `sub_award_types`?

Reply:
I am not that good at predicting the future

Reply:
I don't know either, but the code looks fine so approved.